### DISCUSSION TITLE
Battery status

### DISCUSSION THREAD

**Discussion by kora24** [i3/i3#6032] [Q&A]
Hi everyone,
My battery charging status is always ~95%. I checked with bumblebee-status and it showed 100%.
How can it be fixed?
Thank you.

**Comment by bancobusto** [comment]
Hi, as explained in this post it's likely that i3status is displaying the battery status based on factory settings of your batteries model number, not it's current ability to hold a charge (which can change for a variety of reasons, including mostly age).  

https://bbs.archlinux.org/viewtopic.php?id=168718

Add the following in your i3status config file

battery all {
        format = "%status %percentage %remaining"
        last_full_capacity = true
}

Be aware that by default depending on your distro there may not be a file located at ~/.config/i3status/config, you may need to copy it from /etc/i3status.conf or somewhere similar. The man page for i3status ought to give you a list of several possible locations. 

Hope this helps.

**Comment by kora24** [comment]
It worked! Thank you!

**Reply by bancobusto** [reply]
Great to hear!
